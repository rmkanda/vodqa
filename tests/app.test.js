import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";

import Home from "../src/home/index";

describe("aa", () => {
  test("Snapshot of Body", () => {
    const { asFragment } = render(<Home name="Hello!" />);

    expect(asFragment()).toMatchSnapshot();
  });
});
