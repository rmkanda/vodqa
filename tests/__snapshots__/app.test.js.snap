// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`aa Snapshot of Body 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Welcome to Hello!
    </h1>
    ;
  </div>
</DocumentFragment>
`;
